src/StaticWebAssetsSdk/Targets/Microsoft.NET.Sdk.StaticWebAssets.HtmlImportMap.targets

```diff
@@ -0,0 +1,81 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.StaticWebAssets.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+
+<Project ToolsVersion="14.0">
+    
+  <UsingTask TaskName="Microsoft.AspNetCore.StaticWebAssets.Tasks.WriteImportMapToHtml" AssemblyFile="$(StaticWebAssetsSdkBuildTasksAssembly)" />
+
+    <PropertyGroup>
+    <ResolveBuildRelatedStaticWebAssetsDependsOn>
+      $(ResolveStaticWebAssetsInputsDependsOn);
+      _UpdateHtmlFiles;
+    </ResolveBuildRelatedStaticWebAssetsDependsOn>
+    <ResolveCompressedFilesDependsOn>
+      $(ResolveCompressedFilesDependsOn);
+      _UpdateHtmlFiles
+    </ResolveCompressedFilesDependsOn>
+  </PropertyGroup>
+  <ItemGroup>
+    <StaticWebAssetFingerprintPattern Include="Js" Pattern="*.js" Expression="#[.{fingerprint}]!" />
+    <StaticWebAssetFingerprintPattern Include="Css" Pattern="*.css" Expression="#[.{fingerprint}]!" />
```

Comment:
When we write importmap into the html, we can make a hard fingerprint on user assets, because they are part of the impormap

Reply:
Maybe it should only be `.js`, because for css we can fingerprint only the root files referenced from the html

Reply:
I think if we try to fingerprint CSS and stuff like that it will quickly become a slippery slope (we could do the same thing for images if we wanted to).

I think it is best if we avoid all that and rely on a service worker for those cases (in the end, that's what SWs were defined for).